File: app/src/main/java/com/technolifestyle/therealshabi/brogrammar/MainActivity.java
package com.technolifestyle.therealshabi.brogrammar;

import android.content.Intent;
import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;
import android.view.View;
import android.widget.Button;

import com.technolifestyle.therealshabi.brogrammar.SharedPreferenceUtils.SharedPreferenceStorage;

public class MainActivity extends AppCompatActivity {
    Button mButton;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        if (SharedPreferenceStorage.getSharedPreferenceMainActivity(getBaseContext())) {
            startActivity(new Intent(MainActivity.this, SlackActivity.class));
        }

        mButton = findViewById(R.id.button);


        mButton.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                startActivity(new Intent(MainActivity.this, WebViewActivity.class));
            }
        });

    }

    @Override
    protected void onResume() {
        if (SharedPreferenceStorage.getSharedPreferenceMainActivity(getBaseContext())) {
            startActivity(new Intent(MainActivity.this, SlackActivity.class));
        }
        super.onResume();
    }
}

File: app/src/main/java/com/technolifestyle/therealshabi/brogrammar/MainActivity.kt
package com.technolifestyle.therealshabi.brogrammar

import android.content.Intent
import android.os.Bundle
import android.widget.Button

import androidx.appcompat.app.AppCompatActivity

import com.technolifestyle.therealshabi.brogrammar.sharedPreferenceUtils.SharedPreferenceStorage

class MainActivity : AppCompatActivity() {
    private lateinit var mButton: Button

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        if (SharedPreferenceStorage.getSharedPreferenceMainActivity(baseContext)) {
            startActivity(Intent(this@MainActivity, SlackActivity::class.java))
        }

        mButton = findViewById(R.id.button)


        mButton.setOnClickListener { startActivity(Intent(this@MainActivity, WebViewActivity::class.java)) }

    }

    override fun onResume() {
        if (SharedPreferenceStorage.getSharedPreferenceMainActivity(baseContext)) {
            startActivity(Intent(this@MainActivity, SlackActivity::class.java))
        }
        super.onResume()
    }
}
